### PR TITLE
Update tooltips of the follow us section

### LINKED ISSUE #1346: Tooltips are not working in the footer icons
**Describe the bug**
Currently, the tooltips of the footer icons are not working properly

**To Reproduce**
1. Go to 'sef-site'
3. Scroll down to 'footer'

**Expected behavior**
Fix the tooltips correctly to work

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1346 

## Goals

Fixing the tooltips

## Approach

Fixed the tooltips

### Screenshots

image

  
### Preview Link

https://pr-1366-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by YoshithaRathnayake** [comment]
@anjula-sack @Piumal1999 @jayasanka-sack how about this? Can you review this PR?

File: assets/content/static/footer.html [hunk 1]
```diff
@@ -6,44 +6,36 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
         </div>
         <div class="col-lg-6 text-lg-center btn-wrapper">
             <a target="_blank" href="https://twitter.com/goasksef"
-                class="btn btn-neutral btn-icon-only btn-twitter btn-round btn-lg mt-2" data-toggle="tooltip"
-                data-original-title="Follow us">
+                class="btn btn-neutral btn-icon-only btn-twitter btn-round btn-lg mt-2" data-toggle="tooltip" title="Follow us">
```
Comment (jayasanka-sack) [inline assets/content/static/footer.html]: How about "Follow us on Twitter" ? or just "Twitter"

**Review by jayasanka-sack — COMMENTED** [review]
Good catch @YoshithaRathnayake ! 😍

File: assets/content/static/footer.html (hunk 1 above)
Comment (YoshithaRathnayake) [reply assets/content/static/footer.html]: Yes I thought that @jayasanka-sack but I put the one in the early code. Then shall I put as Twitter?

Comment (YoshithaRathnayake) [reply assets/content/static/footer.html]: > How about "Follow us on Twitter" ? or just "Twitter"

Follow us on Twitter is too long right? @jayasanka-sack Is it ok if it is long?

Comment (YoshithaRathnayake) [reply assets/content/static/footer.html]: I think it is ok with the Follow us on Twitter also @jayasanka-sack

Comment (jayasanka-sack) [reply assets/content/static/footer.html]: let's go with the name. ex: "Twitter", "Facebook", etc.

Comment (YoshithaRathnayake) [reply assets/content/static/footer.html]: Sure @jayasanka-sack I'll update and let you know

Comment (jayasanka-sack) [reply assets/content/static/footer.html]: Thanks @YoshithaRathnayake ! ❤️

File: assets/content/static/footer.html [hunk 2]
```diff
@@ -6,36 +6,36 @@ <h4 class="mb-0 font-weight-light">Get in touch with using any of these platform
...
             </a>
             <a target="_blank"
                 href="https://join.slack.com/t/sefheadquarters/shared_invite/zt-1h5zt3go4-wnRDDpecbWiTdpDv1VUoVg"
-                class="btn btn-neutral btn-icon-only btn-slack btn-round btn-lg mt-2" data-toggle="tooltip" title="Join us">
+                class="btn btn-neutral btn-icon-only btn-slack btn-round btn-lg mt-2" data-toggle="tooltip" title="Slack">
                 <i class="fab fa-slack"></i>
             </a>
             <a target="_blank" href="https://sef.discourse.group"
-                class="btn btn-primary btn-icon-only btn-discourse btn-round btn-lg mt-2" data-toggle="tooltip" title="Join us">
+                class="btn btn-primary btn-icon-only btn-discourse btn-round btn-lg mt-2" data-toggle="tooltip" title="Discourse">
```
Comment (jayasanka-sack) [inline assets/content/static/footer.html]:
```suggestion
                class="btn btn-primary btn-icon-only btn-discourse btn-round btn-lg mt-2" data-toggle="tooltip" title="SEF Hive">
```

Comment (YoshithaRathnayake) [reply assets/content/static/footer.html]: Ok @jayasanka-sack

Comment (YoshithaRathnayake) [reply assets/content/static/footer.html]: Done @jayasanka-sack

**Review by jayasanka-sack — APPROVED** [review]
Thanks @YoshithaRathnayake. We are so glad to have you in the team! ❤️

**Pull request merged**